bundles/org.palladiosimulator.analyzer.slingshot.core/src/org/palladiosimulator/analyzer/slingshot/core/extension/ExtensionIds.java

```diff
@@ -1,9 +1,14 @@
 package org.palladiosimulator.analyzer.slingshot.core.extension;
 
 public final class ExtensionIds {
-	
+
 	public static final String EXTENSION_POINT_ID = "org.palladiosimulator.analyzer.slingshot";
 	public static final String EXTENSION_ELEMENT_NAME = "slingshot-extension";
 	public static final String EXTENSION_ATTRIBUTE_NAME = "module";
-	
+
+
+	public static final String SYSTEM_EXTENSION_POINT_ID = "org.palladiosimulator.analyzer.slingshot.system";
+	public static final String SYSTEM_EXTENSION_ELEMENT_NAME = "slingshot-system-extension";
+	public static final String SYSTEM_EXTENSION_ATTRIBUTE_NAME = "module";
+
```

Comment:
If we do this, maybe it makes sense to also rename the extension point to `slingshot-simulation-extension`?

Reply:
i agree :D